Makefile

```diff
@@ -124,6 +125,27 @@ registry-login:
 build-image:
 	ko publish --base-import-paths --tags="${IMAGE_TAG}" ./cmd/$(IMGCTRL) 
 
+install-helm:
+	mkdir ./helm_bin
+ifeq ($(CUR_OS),Darwin)
+	curl -o ./helm_bin/helm.tar.gz \
+		https://get.helm.sh/helm-v3.8.2-darwin-amd64.tar.gz
+	tar -xf ./helm_bin/helm.tar.gz -C ./helm_bin
+	cp ./helm_bin/darwin-amd64/helm /usr/local/bin
+else 
+	curl -o ./helm_bin/helm.tar.gz \
+		https://get.helm.sh/helm-v3.8.2-linux-amd64.tar.gz
+	tar -xf ./helm_bin/helm.tar.gz -C ./helm_bin
+	cp ./helm_bin/linux-amd64/helm /usr/local/bin
```

Comment:
These `tar` and `cp` commands are equal for both OS, maybe move them outside of the `if` and `else` branches.